keyboards/kinesis/keymaps/tw1t611/config.h

```diff
@@ -0,0 +1,8 @@
+#ifndef CONFIG_USER_H
```

Comment:
The complete contents of this file should be:

```c
#pragma once

// place overrides here
```

This change will make it so the build script only includes this file once, and also removes the redundant inclusion of the Kinesis's keyboard-level header file `keyboards/kinesis/config.h`. QMK's build script will include that file automatically at compile time, so it actually doesn't need to be specified here.

You could actually delete this entire file at this point and suffer no ill effects, because you don't have any overrides set.

Reply:
thank you for clarify